src/pages/mobile-system-defaults/form/mobileSystemDefaultForm.js

```diff
@@ -94,6 +111,46 @@ export default function MobileSystem() {
               maxAccess={access}
             />
           </Grid>
+          <Grid item xs={12} sx={{ marginLeft: '0.5rem', marginRight: '0.5rem' }}>
+            <CustomTextField
+              name='rt_mob_whatsapp'
+              label={labels.whatsapp}
+              value={formik.values.rt_mob_whatsapp}
+              maxAccess={access}
+              onChange={formik.handleChange}
+              onClear={() => formik.setFieldValue('rt_mob_whatsapp', '')}
+            />
```

Comment:
why there isn't any error check in any of the new fields?

Reply:
not mandatory